Kirill_Kazimirov/02.Git.Local/README.md

```diff
@@ -0,0 +1,135 @@
+#Homework Assignment 1: Initializing a Local Repository
```

Comment:
You need to add white-space after #

please correct all headers

Reply:
Fix it